src_c/camera_v4l2.c

```diff
@@ -177,8 +177,8 @@ v4l2_xioctl(int fd, int request, void *arg)
    currently two step processes. */
 /* TODO: Write single step conversions where they may actually be useful */
 int
-v4l2_process_image(pgCameraObject *self, const void *image,
-                   unsigned int buffer_size, SDL_Surface *surf)
+v4l2_process_image(pgCameraObject *self, const void *image, int buffer_size,
```

Comment:
A size really only makes sense as an unsigned int, because a size can't be negative. But I don't see images getting that big.

Reply:
Yeah I remember the compiler crying about stuff here, and if I left `size` as an unsigned int, I'd have to do more changes around the code to make the compiler happy. This was the easier solution, but if it has the be other way, then I'm okay for doing it that way too

Reply:
Cool, let's leave it as it is now then.